Classes/Mvc/RequestBuilder.php

```diff
@@ -29,17 +29,16 @@
 
 use Helhum\TyposcriptRendering\Exception;
 
-/**
- * Class RequestBuilder
- */
 class RequestBuilder
 {
     /**
-     * @param array $rawRequestArgument
+     * @param mixed[] $rawRequestArgument
+     *
      * @return Request
+     *
      * @throws \Helhum\TyposcriptRendering\Exception
      */
-    public function build($rawRequestArgument)
+    public function build(array $rawRequestArgument)
```

Comment:
If $rawRequestArgument is something other than an array, the code will break in the first line of the method anyway with a warning:
    if (empty($rawRequestArgument['context']) || !is_string($rawRequestArgument['context'])) {

I'd prefer the interpreter warning about the cause of the breakage then instead of the symptom.